### DISCUSSION TITLE
https支持开启后，socket：error on * Connection reset by peer

### DISCUSSION THREAD

**Discussion by BonneauYang** [cloudwu/skynet#1645] [General]
1.根据Wiki上关于支持https的说明文档，修改**makefile**文件**重新编译**了skynet。
2.增加skynet启动配置**enablessl**为true。
3.使用postman工具向服务器发送数据。
4.根据示例simpleweb脚本编写处理https相关行为的从服务（**gen_interface**函数的**protocol**参数设为**https**）。
5.在**socket.start(fd)**之后，调用**gen_interface**函数获取数据处理接口interface，调用interface.init时，**tlshelper.lua第31行writefunc(ds2)，追溯至sockethelper.lua第65行writefunc函数报错SocketError**！
有木有哪位大佬指点一下呐，谢谢！！！
skynet打印信息socket：error on * Connection reset by peer

**Comment by cloudfreexiao** [comment]
这和 skynet 没关系 看看这个的介绍
https://stackoverflow.com/questions/1434451/what-does-connection-reset-by-peer-mean

**Comment by BonneauYang** [comment]
> 这和天网没关系 看看这个的介绍 [https://stack overflow . com/questions/1434451/connection-reset-by-peer是什么意思](https://stackoverflow.com/questions/1434451/what-does-connection-reset-by-peer-mean)
好的，我看看，谢谢~
